### ISSUE TITLE
object detection training loss increasing 

### ISSUE THREAD

**Issue by pandasMX** [tensorflow/models#3637]
Hi, I first trained a 20 classes SSD mobilenet model and it reached a mAP of 0.8. 
However, now when I train a 100 classes model, I observed the training loss going up after 200k steps. 
I did not change any code, and the code for producing training data is the same as before. I edited the config file and changed the num class and all the paths to training data and label map. I also added a few more data augmentation. 

<img width="1034" alt="screen shot 2018-03-18 at 9 45 46 am" src="https://user-images.githubusercontent.com/32104513/37561739-3680bf66-2a91-11e8-93d5-a1a8cdc2b39f.png">

Is it my learning rate 0.004 too high? Or any other possible reasons can cause this? Thanks

**Comment by bnjff** [comment]
Your problem is likely due to a bug in data augmentation: https://github.com/tensorflow/models/issues/2066

The fix is in this pull request that hasn't been merged: https://github.com/tensorflow/models/pull/2146

Also. in your config, you'll need a new data_augmentation_options key for each option:

```
  data_augmentation_options {
    random_horizontal_flip {}
  }
  data_augmentation_options {
    random_vertical_flip{
    }
  }
```

**Comment by wt-huang** [comment]
Closing this as the PR has been merged.

**Comment by Siggi1988** [comment]
Hallo pandasMX, 

is the loss in percent?
